packages/ui/components/calendar/components/currentTime/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from "react";
+
+import dayjs from "@calcom/dayjs";
+
+import { useCalendarStore } from "../../state/store";
+
+type Props = {
+  containerNavRef: React.RefObject<HTMLDivElement>;
+  containerRef: React.RefObject<HTMLDivElement>;
+  containerOffsetRef: React.RefObject<HTMLDivElement>;
+};
+
+export function CurrentTime({ containerOffsetRef }: Props) {
+  const [currentTimePos, setCurrentTimePos] = useState<number>(0);
+  const { startHour, endHour } = useCalendarStore((state) => ({
+    startHour: state.startHour || 0,
+    endHour: state.endHour || 23,
+  }));
+
+  useEffect(() => {
+    // Set the container scroll position based on the current time.
+    let currentMinute = new Date().getHours() * 60;
+    currentMinute = currentMinute + new Date().getMinutes();
+
+    if (containerOffsetRef.current) {
+      const totalHours = endHour - startHour;
+      const currentTimePos = currentMinute;
+      setCurrentTimePos(currentTimePos);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [startHour, endHour]);
+
+  return (
+    <div
+      className="absoulte z-40 ml-3 flex h-1 items-center justify-center border-gray-800"
```

Comment:
1. There is a typo in `absolute`
2. Should we make this `aria-hidden`, since I can imagine this time label might only confuse visually impaired users? 🤔  I don't think it will bring any value to them either.

Reply:
Very true